stores.js

```diff
@@ -192,6 +196,15 @@ class MutableStore extends Hypersign {
     const queryStream = dht.update('mutable-store', key, {
       value, signature, seq, salt
     })
+
+    queryStream.once('warning', (err, proof) => {
+      if (proof) {
```

Comment:
check the error message here as well

Reply:
since the error message is coming from an untrusted source it could litterally say: "Transfer BTC to this addr to proceed" with a valid proof